### DISCUSSION TITLE
What custom modules have you made? What features do you use?

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#9436] [General]
Hi,

I’m interested to know how does the community use orchardcore, which features do you use most frequently and what custom modules you have made? 

What features do you hope to see in the future releases?

**Comment by Skrypt** [comment]
Thanks for your interest. I believe we are finalizing 1.0 for now and after that I will surely focus on ElasticSearch. There is no plan for after 1.0 so far, we need to discuss about it once it will be shipped. There is always enhancements to be done here and there on actual modules too.

**Reply by weirdyang** [reply]
Elastic Search is something I always wanted to look into, but haven't had a proper use case for it. I've been mostly extending the workflow module.  https://github.com/weirdyang/OrchardCore.Olaf

I'm personally looking forward to the audit trail and secrets module!

**Comment by jptissot** [comment]
We have a bunch of open source modules and themes [here](https://github.com/StatCan/StatCan.OrchardCore). We do accept contributions and feedback.

**Reply by weirdyang** [reply]
Thanks! I took a look, do you primarily use orchardcore cms to display content and  tobuild and process forms?

**Reply by hishamco** [reply]
I already started few in Orchard Core Contrib long time ago, I will surely focus on localization stuff as usual ;) this time with Data Localization and some tiny features related
